bindings/python/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "tokenizers-python"
 version = "0.11.0"
 authors = ["Anthony MOI <m.anthony.moi@gmail.com>"]
-edition = "2018"
+edition = "2021"
```

Comment:
@Narsil if you cherry-pick [8691345](https://github.com/huggingface/tokenizers/pull/1066/commits/8691345d733b40c3923b09d67b7e795024993e06) onto your branch for #1064 I think it may fix the issue.

The thinking is thatedition 2021 uses Cargo's newer `resolver = "2"` which won't add the `auto-initialize` feature from the pyo3 dev-dependency during the manylinux build, which I think will avoid the problem.